tests/dashbio_demos/app_molecule3d.py

```diff
@@ -46,17 +47,90 @@ def layout():
                 children=[
                     dcc.Tabs([
                         dcc.Tab(
-                            label='What is Molecule 3D?',
+                            label='About',
                             value='what-is',
                             children=html.Div(className='mol3d-tab', children=[
-                                html.P('Molecule3D allows you to view ' +
-                                       'biomolecules in different ' +
-                                       'representations -- sticks, ' +
-                                       'spheres, and cartoons.')
+                                html.H4('What is Molecule3D?'),
+                                html.P('Molecule3D is a visualizer that allows you '
+                                       'to view biomolecules in multiple representations: '
+                                       'sticks, spheres, and cartoons.'),
+                                html.P('You can choose a preloaded structure, or upload your own, '
+                                       'in the "Data" tab. A sample structure is also '
+                                       'available to download.'),
+                                html.P('In the "View" tab, you can change the style and '
+                                       'coloring of the various components of each molecule.')

```

Comment:
```suggestion
                                       'coloring of the various components of your molecule.')
```
'each' may imply that we're viewing more than one molecule at a time.